pkg/ephemeral/io/carrier_test.go

```diff
@@ -166,4 +194,42 @@ var _ = Describe("Carrier", func() {
 			Expect(err).To(HaveOccurred())
 		})
 	})
+
+	Context("when connecting as Player0", func() {
+		playerId := int32(0)
+		It("will receive and handle the server's fileHeader", func() {
+			// Arrange
+			// ToDo: Better Response for real-life scenario?
+			serverResponse := []byte{1, 0, 0, 0, 1} // 4 byte length + header, in this case "1". In real case Descriptor + Prime
```

Comment:
What is the exact format of the prime transmitted?

Reply:
See here for the Header

https://github.com/data61/MP-SPDZ/issues/418#issuecomment-975424591

    It is made up as follows:

    - length to follow (29 as little-endian 8-byte number)
    - protocol descriptor ("SPDZ gfp")
    - serialization of the prime: sign bit (0 as 1 byte), 
      length to follow (16 as little-endian 4-byte number), 
      prime (170141183460469231731687303715885907969 in big-endian)
    
    The next version will add general documentation of this.

Reply:
So, should we change this to match a "real-life" server response, or are we fine keeping this stub value?

Reply:
Fine for me as well.